crates/driver/src/infra/solver/mod.rs

```diff
@@ -289,8 +289,14 @@ impl Solver {
             started_at.elapsed(),
         );
         let res = res?;
-        let res: solvers_dto::solution::Solutions = serde_json::from_str(&res)
-            .tap_err(|err| tracing::warn!(res, ?err, "failed to parse solver response"))?;
+        let res: solvers_dto::solution::Solutions = serde_json::from_str(&res).tap_err(|err| {
+            tracing::warn!(res, ?err, "failed to parse solver response");
+            self.notify(
+                auction.id(),
+                None,
+                notify::Kind::SerializationError("Error: request format invalid".to_string()),
```

Comment:
To make this useful for solvers we should send the `serde` error message as well. That includes the actual thing that tripped up the deserializer (e.g. missing field, value with wrong format, etc.).
Also I think we don't need the `Error:`, right?